dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -146,20 +146,37 @@ const hideIconBelowLeftCol = css`
 	}
 `;
 
+const pictureRatio = (13 / 18) * 100;
+const videoRatio = (23 / 36) * 100;
+
 const iconStyle = (palette: Palette) => css`
 	fill: ${palette.fill.cameraCaptionIcon};
 	margin-right: ${space[1]}px;
 	display: inline-block;
-	vertical-align: middle;
+	position: relative;
+	width: 1em;
+	vertical-align: baseline;
```

Comment:
As mentioned by @JamieB-gu, it’s worth checking all browsers respect this in the same way!

Reply:
There’s [full support for `vertical-align`, according to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align).